src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -72,7 +74,7 @@
 @Slf4j
 @RequestMapping(value = "secure", produces = MediaType.APPLICATION_JSON_VALUE)
 @AllArgsConstructor
-@PreAuthorize("hasRole('ROLE_COMMUNITY')")
+@PreAuthorize("hasAnyRole('ROLE_COMMUNITY','ROLE_COMMUNITY_API_OPEN','ROLE_COMMUNITY_API_EXCLUDED','ROLE_COMMUNITY_API_EXCLUDED_RESTRICTED','ROLE_COMMUNITY_API_RESTRICTED')")
```

Comment:
Could apply this only on the relevant methods if that's preferable

Reply:
Do we still need this at all given we are using scope?